server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/functions.js

```diff
@@ -460,6 +456,7 @@ function clearTableProblems(tableID) {
   call = sanitize(call);
   // make the rest call, passing success function callback
   $.post(call, function () {
+    console.info("REST POST call to " + call);
     refreshProblems();
   });
```

Comment:
I think this pattern shows up a couple of times and could be put into it's own function (the following is only pseudo-code, so I don't expect it to work as-is):

```javascript
function doLoggedPostCall(endpoint, callback) {
  console.info("POST call to " + endpoint);
  $.post(endpoint, function() {
    console.debug("POST call to " + endpoint + " : success"); // not sure the appropriate message here
    if (callback != null) {
      // execute callback; for example, refreshProblems() in this case... could be something else in other cases.
    }
  }
}
```

Then, This becomes:
```javascript
  doLoggedPostCall(call, refreshProblems);
```